### PR TITLE
Update manual test plan

### LINKED ISSUE #2: Add .editorconfig

### LINKED ISSUE #3: Set the file type to test plan files

### LINKED ISSUE #4: Add more tests to "data"
We actually have not thoroughly tested with a bunch of data. Travis clears the profile folder every time when automated tests are conducted, so automated tests have not covered that area.

To avoid betraying early our early adopters' trust, we should focus more on testing with the profile folder full of the data.

### LINKED ISSUE #9: Replace safebrowsing URL
from downloadme.org to https://www.raisegame.com/

### PR DESCRIPTION
**Pull request by luixxiul**
- Add .editorconfig (same as used on brave/browser-laptop)

Closes #2
Closes #3
Closes #4
Closes #9

### CODE REVIEW

**Comment by luixxiul** [comment]
CC @brave/qa

File: wikitemplate.md [hunk 1]
```diff
@@ -138,12 +155,15 @@
 8. [ ] Open an email on http://mail.google.com/ or inbox.google.com and click on a link. Make sure it works.
 9. [ ] Test that PDF is loaded at http://www.orimi.com/pdf-test.pdf
 10. [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).
+11. [ ] Test that extension icons can be displayed on the navigation bar
 
 ## Flash tests
+
 1. [ ] Turn on Flash in about:preferences#security. Test that clicking on 'Install Flash' banner on myspace.com shows a notification to allow Flash and that the banner disappears when 'Allow' is clicked.
```
Comment (srirambv) [inline wikitemplate.md]: This is also obselete. It now shows Cookie banner on the site. Can this be removed?

**Comment by luixxiul** [comment]
why wouldn't we address that with another PR?

File: wikitemplate.md (hunk 1 above)
Comment (luixxiul) [reply wikitemplate.md]: Let's open an issue for that :-)

**Comment by luixxiul** [comment]
> Not sure if this test is required https://extensions.inrialpes.fr/brave/ . Site shows as Chrome irrespective of Shields settings and plugin enabled/disabled

This site had once detected Brave, so it should be kept I think.

**Comment by luixxiul** [comment]
Rebased.

**Pull request merged**